xsnippet/api/database.py

```diff
@@ -28,19 +28,7 @@ def create_connection(conf):
     mongo = AsyncIOMotorClient(conf['DATABASE_CONNECTION_URI'])
 
     # get_default_database returns a database from the connection string
-    db = mongo.get_default_database()
-
-    # ID incrementer is used to auto increment record ID if nothing
-    # is explicitly passed.
-    #
-    # ID processor is used for auto converting domain IDs to and from
-    # database ones (i.e. 'id' -> '_id' and vice versa).
-    #
-    # SON manipulators are applied in reversed order.
-    db.add_son_manipulator(_IdIncrementer())
```

Comment:
They have finally removed SON manipulators, hah. `_id` trickery  is a bit annoying, but we can revisit this later.

Reply:
> They have finally removed SON manipulators

Yeah. Now I think it was a bad idea. Even though it was mine :P